final-project/scripts/main.js

```diff
@@ -1,10 +1,101 @@
 import "../styles/main.scss";
 
-// testing that `const` and arrow functions transpile correctly
-const before = "Hello -world!";
-const after = before
-  .split("")
-  .filter(char => char !== "-")
-  .join("");
-console.log({ before });
-console.log({ after });
+const articlesUrl = 'https://wt-4662f45b9eefda7172b747b28d23efdb-0.sandbox.auth0-extend.com/blog'; 
+
+getArticlesFromUrl();
+
+async function getArticlesFromUrl(){
+  fetch(articlesUrl)
+  .then(resp => resp.json())
+  .then(function(data) {
+    loadCarousel(data.articles)
+  });
+}
+
+function loadCarousel(articles){
+  articles.forEach((article, index) => {
+    // Create image
+    var imageDiv = document.createElement('div');
+    imageDiv.setAttribute('class', 'blog__content__articles__cards__content__image');
```

Comment:
Probably it would have been more readable and maintainable to have these divs cloned and transformed from the HTML instead of creating them from scratch with JS, just in case you need to change their structure or classes.